conans/test/integration/build_requires/test_relocatable_toolchain.py

```diff
@@ -0,0 +1,84 @@
+import textwrap
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient
+
+
+def test_relocatable_toolchain():
```

Comment:
A docstring explaining the test and a link to the issue that closes would be great.

Reply:
Done